massdriver.yaml

```diff
@@ -121,7 +121,8 @@ params:
       title: Data Redundancy
       type: object
       required:
-        - replication_type
+        - zone_redundancy
```

Comment:
Having `replication_type` here forced the user to selected it, even with `zone_redundancy` disabled.

Reply:
https://user-images.githubusercontent.com/651833/214141375-f3132777-4752-45ed-865f-a5fb321a8aef.mov

Reply:
How it works now

https://user-images.githubusercontent.com/651833/214142233-0f4a32ad-77f5-4195-a612-a15a9251bfcc.mov